src/test/rpc/GatewayBalances_test.cpp

```diff
@@ -148,6 +148,36 @@ class GatewayBalances_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testGWBApiV2(FeatureBitset features)
+    {
+        using namespace std::chrono_literals;
+        using namespace jtx;
+        Env env{
+            *this,
+            envconfig([](std::unique_ptr<Config> c) {
+                c->loadFromString("\n[beta_rpc_api]\n1\n");
+                return c;
+            }),
+            features};
```

Comment:
Since #4573 is merged, this can probably be changed to Env env{*this}; ?

Reply:
done 👍

Reply:
there's more over here: https://github.com/XRPLF/rippled/pull/4577#discussion_r1239092997